### PR TITLE
feat: using built in ttl

### PR DESCRIPTION
**Pull request by jleon15**
## Description

-

## Testing required outside of automated testing?

- [ ] Not Applicable

### Screenshots (if appropriate):

- [ ] Not Applicable

## Rollback / Rollforward Procedure

- [ ] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: cypress/specs/acceptance/pages/api/secrets/createSecret.spec.ts [hunk 1]
```diff
@@ -5,9 +5,17 @@ describe('create secret', () => {
...
   beforeEach(() => {
     secretData = chance.guid();
-    secretTtl = chance.integer();
+    secretTtl =
+      ttls[
+        chance.integer({
+          min: 0,
+          max: 2,
+        })
+      ];
```
Comment (brigonzalez) [inline cypress/specs/acceptance/pages/api/secrets/createSecret.spec.ts:18]:
```suggestion
      chance.pickone(ttls);
```

File: pages/api/secrets/[id]/details.ts [hunk 2]
```diff
@@ -24,19 +24,12 @@ export default async (
...
-        id: token.id,
-        timeLeft: datePlusTTL,
-      });
-    } else {
-      await bt.tokens.delete(id);
-      res.status(404).json({});
-    }
+    res.status(200).json({
+      id: token.id,
+      timeLeft: expiresAt,
```
Comment (brigonzalez) [inline pages/api/secrets/[id]/details.ts:31]: are we actually using this? it doesn't look like we are. if we are, we would need to calculate the actual time left here

File: docker-compose-wiremock.yml [hunk 3]
```diff
@@ -6,4 +6,6 @@ services:
     ports:
       - '8080:8080'
       - '8443:8443'
-    command: [ '--https-port', '8443' ]
+    command:
+      - --https-port=8443
+      - --verbose
```
Comment (brigonzalez) [inline docker-compose-wiremock.yml:11]: is this useful? or accidentally left over?

File: pages/api/secrets/[id]/details.ts (hunk 2 above)
Comment (jleon15) [reply pages/api/secrets/[id]/details.ts]: we weren't idk why it was there, removed it

File: docker-compose-wiremock.yml (hunk 3 above)
Comment (jleon15) [reply docker-compose-wiremock.yml]: it logs more things on the wiremock container, i added it for troubleshooting but not really needed, removed it

**Review by brigonzalez — APPROVED** [review]

**Pull request merged**